src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -29,25 +29,20 @@ case class BasicProfile (
     title: String,
     contactEmail: Option[String],
     institute: String,
-    institutionalProgram: String,
     programLocationCity: String,
     programLocationState: String,
     programLocationCountry: String,
-    pi: String,
-    nonProfitStatus: String,
-    termsOfService: Option[String]
+    termsOfService: Option[String],
```

Comment:
This `termsOfService` stuff seems to be unused from an earlier implementation of ToS in Sam.

I plan to verify that and rip it out but not as part of this PR